modules/types/src/schemas/engine.ts

```diff
@@ -167,6 +167,17 @@ const WithdrawParamsSchema = Type.Object({
   initiatorSubmits: Type.Optional(Type.Boolean()),
 });
 
+// Run Auction Params
+
+const RunAuctionParamsSchema = Type.Object({
```

Comment:
to be reviewed: should params be mandatory or optional ?

Reply:
Let's make all of them mandatory just to make things easier.

Reply:
changed to mandatory